feeds.yml

```
@@ -157,4 +157,9 @@ rednafi:
   title: rednafi.com
   scrape_url: https://rednafi.com/go/index.xml
   url: https://rednafi.com/go
-  scraper: FeedScraper
\ No newline at end of file
+  scraper: FeedScraper
+jvt:
+  title: Jamie Tanna | Software Engineer
+  scrape_url: https://www.jvt.me/tags/go/feed.xml
```

I've been meaning to create an articles-only feed page for tags, so this has been a good nudge to get into doing it!

I've just pushed the changes, which should mean you can now pull:

```suggestion
  scrape_url: https://www.jvt.me/tags/go/feed.articles.xml
```

And this will only list blog posts.

This will be live in ~10-20 minutes ([build+deploy job](https://gitlab.com/tanna.dev/jvt.me/-/pipelines/1251096602)) at which point we should be good to merge this!

---

That's super cool of you @jamietanna  😄  thanks

@tomasurbanec Let me know what you think, thanks

---

Hi,
sorry for long time without response.
I will happily merge it. Thanks, both of you, for thinking it through so well. :+1: